tools/proto-js-plugin/src/main/java/io/spine/js/generate/CodeLine.java

```diff
@@ -30,11 +34,21 @@
  * <p>For example, the code inside the {@code if} block is one unit deeper than the {@code if}
  * declaration itself.
  */
-final class CodeLine {
+public final class CodeLine {
```

Comment:
Let's make this class abstract and create descendants for specific cases. Currently, these cases are processed in `CodeLined` class using `String + String` operator. What I'm suggesting are classes, such as `Comment` and `VariableDeclaration`.